### PR TITLE
Ruby: fix credentials for longrunning and tests for clients with no major version

### PR DESCRIPTION
**Pull request by landrito**

### CODE REVIEW

File: src/main/java/com/google/api/codegen/ruby/RubyUtil.java [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+ */
+package com.google.api.codegen.ruby;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.regex.Pattern;
+
+public class RubyUtil {
+  private static final String LONGRUNNING_PACKAGE_NAME = "Google::Longrunning";
+  private static final String RUBY_PACKAGE_SEPERATOR = "::";
```
Comment (geigerj) [inline src/main/java/com/google/api/codegen/ruby/RubyUtil.java]: Aside: I'm pretty surprised MVVM doesn't provide a special construct for this. (No action required.)

File: src/main/java/com/google/api/codegen/viewmodel/testing/ClientInitParamView.java [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.api.codegen.viewmodel.testing;
+
+import com.google.auto.value.AutoValue;
+import javax.annotation.Nullable;
+
+@AutoValue
+public abstract class ClientInitParamView {
```
Comment (geigerj) [inline src/main/java/com/google/api/codegen/viewmodel/testing/ClientInitParamView.java]: Class-level comments, please.

File: src/main/java/com/google/api/codegen/ruby/RubyUtil.java (hunk 1 above)
Comment (landrito) [inline src/main/java/com/google/api/codegen/ruby/RubyUtil.java]: Ah you're right. Updated.

File: src/main/java/com/google/api/codegen/viewmodel/testing/ClientInitParamView.java (hunk 2 above)
Comment (landrito) [inline src/main/java/com/google/api/codegen/viewmodel/testing/ClientInitParamView.java]: Done.

**Pull request merged**